translations/introduction.tex/es.po

```diff
@@ -26,7 +26,7 @@ msgstr ""
 #. type: Plain text
 #: sections/introduction.tex:3
 msgid "\\addsection{Introduction}{\\spells/magic_arrow.png}"
-msgstr "\\addsection{Introduccion}{\\spells/magic_arrow.png}"
+msgstr "\\addsection{Introducción}{\\spells/magic_arrow.png}"
```

Comment:
```suggestion
msgstr "\\addsection{Introducción}{\\spells/magic_arrow.png}\n\\vspace*{\\lang_header_adjustment}"
```
I think Spanish will need this for every section header. Text seems to start lower than for other languages without it.